MaterialDesignThemes.Wpf/PackIcon.cs

```diff
@@ -17,9 +17,11 @@ public class PackIcon : PackIconBase<PackIconKind>
         static PackIcon()
         {
             DefaultStyleKeyProperty.OverrideMetadata(typeof(PackIcon), new FrameworkPropertyMetadata(typeof(PackIcon)));
-        }     
+        }
 
         public PackIcon() : base(PackIconDataFactory.Create)
-        { }    
+        {
+            FlowDirection = FlowDirection.LeftToRight;
```

Comment:
Yeah, delete it, add the new code in, commit, push, all on the same branch.  The beauty of code reviews with GitHub...you can just keep working on this PR until everything is code reviewed, then I will merge it.